src/SFGUI/Scrollbar.cpp

```diff
@@ -92,6 +92,26 @@ sf::Vector2f Scrollbar::CalculateRequisition() {
 	return sf::Vector2f( mimimum_slider_length, mimimum_slider_length );
 }
 
+void Scrollbar::HandleMouseWheelEvent(int delta, int x, int y) {
+	if (GetOrientation() == Orientation::VERTICAL) {
+		auto slider_rect = GetSliderRect();
+		auto alloc = GetAllocation();
+		sf::FloatRect rect(0, 0, alloc.left + slider_rect.width, alloc.height);
```

Comment:
We like to initialize floating point values with floating point literals (.f suffix).